ibis/expr/types/relations.py

```diff
@@ -2859,6 +2859,98 @@ def info(self) -> Table:
             aggs.append(agg)
         return ibis.union(*aggs).order_by(ibis.asc("pos"))
 
+    def describe(
+        self, quantile: Sequence[ir.NumericValue | float] = [0.25, 0.5, 0.75]
```

Comment:
Do not use a mutable data structure like a `list` or a `dict` as default argument. If that argument gets mutated _anywhere_ the mutation is reflected in the caller.

Here's an example:

```
In [1]: def do_stuff(x: list[int]):
   ...:     x.append(42)

In [2]: my_list = [1, 2, 3]

In [3]: do_stuff(my_list)

In [4]: my_list
Out[4]: [1, 2, 3, 42]

In [5]: do_stuff(my_list)

In [6]: my_list
Out[6]: [1, 2, 3, 42, 42]
```

In this case you can use a `tuple`, since that's an immutable sequence.

```suggestion
        self, quantile: Sequence[ir.NumericValue | float] = (0.25, 0.5, 0.75)
```